packages/cli/src/util/dns/get-domain-dns-records.ts

```diff
@@ -8,29 +8,32 @@ type Response = {
   pagination?: PaginationOptions;
 };
 
+type getDomainDNSRecordsArgs = {
+  output: Output;
+  client: Client;
+  domain: string;
+  nextTimestamp?: number;
+  limit?: number;
+};
+
 export default async function getDomainDNSRecords(
-  output: Output,
-  client: Client,
-  domain: string,
-  apiVersion = 3,
-  nextTimestamp?: number,
-  limit = 20
+  args: getDomainDNSRecordsArgs
 ) {
-  output.debug(`Fetching for DNS records of domain ${domain}`);
+  args.output.debug(`Fetching for DNS records of domain ${args.domain}`);
   try {
-    let url = `/v${apiVersion}/domains/${encodeURIComponent(
-      domain
-    )}/records?limit=${limit}`;
+    let url = `/v4/domains/${encodeURIComponent(args.domain)}/records?limit=${
```

Comment:
Note: that the entry point has the domains set to `v4` here: https://github.com/vercel/vercel/pull/9246/files#diff-91299e67aed9eb69236ea2a90f38d7a8c9a55d45bdf9fb9dfd7d3d8544de93d8L54. But if not set, goes back to `v3` which seems odd. Unless there's a good reason too I think its easier to not have to pass around the `apiVersion` and pin it to a version.